docs/applications/deploy-application-using-incoming-hook.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Deploy Application using an Incoming Hook
+intro: As project users base grow, there's a need to rely more on processes and automation; Devopness helps by providing, among other tools, customizable CI/CD pipelines, empowering our users to automate common and repetitive tasks to improve code quality and their code review processes. Create a incoming webhook to trigger a application deploy programmatically.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+Before creating our application deploy incoming webhook, we need the `Application ID` (`<application_id>`) and `Deploy Pipeline ID` (`<pipeline_id>`) to use as parameters in our API calls.
+
+<details open>
+  <summary>Steps to find `Application ID` and `Deploy Pipeline ID`</summary>
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+1. In the list of applications, find the application with the pipeline you want to manage and click `DETAILS`
+1. On the upper-right corner of the applications details view click `SETTINGS`
+1. Use the drop-down menu to choose `PIPELINES`
+1. In the list of pipelines, find the pipeline you want to manage and click `DETAILS`
+    > If you haven't created a pipeline yet, follow the [Add a Pipeline](/docs/pipelines/add-pipeline) guide
+1. Copy the `Application ID` and `Deploy Pipeline ID` from the URL, considering the following URL format:
+    ```bash
+    https://app.devopness.com/projects/<project_id>/environments/<environment_id>/applications/<application_id>/pipelines/<pipeline_id>
+    ```
+
+</details>
+
+Once you have your `Application ID` (`<application_id>`) and `Deploy Pipeline ID` (`<pipeline_id>`), please follow the instructions below to add an incoming webhook to your application that triggers the deploy pipeline:
+
+1. Create an incoming webhook using the payload below, replacing `<application_id>` and `<pipeline_id>`
+    > Follow the [Create an Incoming Webhook](/docs/webhooks/create-incoming-webhook) guide
```

Comment:
I stick to my point in previous comments. Given that:
1. The reader already obtained `<application_id>` and `<pipeline_id>`
2. This topic is already pointing the reader to the `create-incoming-webhook` topic
3. The payload is being exemplified and explained there

Why do we need to repeat the payload example here?

Reply:
@jfoliveira currently both posts are using similar but different payloads, being the one on the create incoming hook simpler; I could replace the payload there using this one.

I believe this would solve the matter being discussed here.

Will update the PR soon.